_config.yml

```diff
@@ -148,9 +146,9 @@ exclude:
   - css
   - pa11y-results.json
 
-env: dev
-
-url: https://designsystem.digital.gov
+# Jekyll SEO tag
 twitter:
   username: USWDS
   card: summary_large_image
```

Comment:
Re-ordered and added comment to better contain SEO plugin specific settings. Hoping it will improve clarity for other devs.